app/components/satis/dropdown/component.html.slim

```diff
@@ -12,7 +12,7 @@ div.satis-dropdown data-action="keydown->satis-dropdown#dispatch" data-controlle
         .h-12.p-1.bg-white.dark:bg-gray-900.opacity-25.flex.border.border-gray-200.opacity-25.dark:border-gray-600.rounded.opacity-75
           .flex.flex-auto.flex-wrap
             / Input where you can search
-            input.p-1.px-2.appearance-none.outline-none.w-full.text-gray-800.dark:bg-gray-800.dark:text-gray-300 data-action="input->satis-dropdown#search" data-satis-dropdown-target="searchInput" placeholder=placeholder autofocus=options[:autofocus]
+            input.p-1.px-2.appearance-none.outline-none.w-full.dark:bg-gray-800.dark:text-gray-300 data-action="input->satis-dropdown#search" data-satis-dropdown-target="searchInput" placeholder=placeholder autofocus=options[:autofocus]
```

Comment:
Add a screenshot of how this will look please, it should still fit with what we have.